### PR TITLE
Docs: Clarify what should be comitted to source control

### PR DESCRIPTION
**Pull request by jaredbeck**
### Motivation

After sorbet has been installed, and we confirm that everything is working, the *very next* question is what to commit to source control. Every person installing sorbet will have this question.

In this PR, I make a *guess* about what should be committed.

### Test plan

N/A

### CODE REVIEW

File: website/docs/adopting.md [hunk 1]
```diff
@@ -161,6 +161,11 @@ enable. The tl;dr is that type checking happens when we add
 [`# typed: true`](static.md) to files and write [method signatures](sigs.md) in
 those files.
 
+## Step 6: Source Control
+
+All files generated by `srb init` should be committed to source control, 
+including the `Makefile` and the entire `sorbet` directory.
```
Comment (ptarjan) [inline website/docs/adopting.md]: there shouldn't be a `Makefile`, only the `sorbet` directory. Maybe one of your files made that?

**Comment by jaredbeck** [comment]
> there shouldn't be a Makefile, only the sorbet directory. Maybe one of your files made that?

Reference to `Makefile` removed. It's a mystery to me too!

**Comment by DarkDimius** [comment]
kicked of CI

**Comment by DarkDimius** [comment]
@jaredbeck could you run `./tools/scripts/format_website.sh` to reformat your changes?

**Comment by connorshea** [comment]
Huh, mine made a Makefile as well! @jaredbeck what does your gemfile.lock look like?

<details>
<summary>Gemfile.lock</summary>

```
GEM
  remote: https://rubygems.org/
  specs:
    actioncable (5.2.3)
      actionpack (= 5.2.3)
      nio4r (~> 2.0)
      websocket-driver (>= 0.6.1)
    actionmailer (5.2.3)
      actionpack (= 5.2.3)
      actionview (= 5.2.3)
      activejob (= 5.2.3)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 2.0)
    actionpack (5.2.3)
      actionview (= 5.2.3)
      activesupport (= 5.2.3)
      rack (~> 2.0)
      rack-test (>= 0.6.3)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (5.2.3)
      activesupport (= 5.2.3)
      builder (~> 3.1)
      erubi (~> 1.4)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.3)
    active_storage_validations (0.7.1)
      rails (>= 5.2.0)
    activejob (5.2.3)
      activesupport (= 5.2.3)
      globalid (>= 0.3.6)
    activemodel (5.2.3)
      activesupport (= 5.2.3)
    activerecord (5.2.3)
      activemodel (= 5.2.3)
      activesupport (= 5.2.3)
      arel (>= 9.0)
    activestorage (5.2.3)
      actionpack (= 5.2.3)
      activerecord (= 5.2.3)
      marcel (~> 0.3.1)
    activesupport (5.2.3)
      concurrent-ruby (~> 1.0, >= 1.0.2)
      i18n (>= 0.7, < 2)
      minitest (~> 5.1)
      tzinfo (~> 1.1)
    addressable (2.6.0)
      public_suffix (>= 2.0.2, < 4.0)
    arel (9.0.0)
    ast (2.4.0)
    aws-eventstream (1.0.3)
    aws-partitions (1.162.0)
    aws-sdk-core (3.52.1)
      aws-eventstream (~> 1.0, >= 1.0.2)
      aws-partitions (~> 1.0)
      aws-sigv4 (~> 1.1)
      jmespath (~> 1.0)
    aws-sdk-kms (1.20.0)
      aws-sdk-core (~> 3, >= 3.52.1)
      aws-sigv4 (~> 1.1)
    aws-sdk-s3 (1.39.0)
      aws-sdk-core (~> 3, >= 3.52.1)
      aws-sdk-kms (~> 1)
      aws-sigv4 (~> 1.0)
    aws-sigv4 (1.1.0)
      aws-eventstream (~> 1.0, >= 1.0.2)
    bcrypt (3.1.13)
    better_errors (2.5.1)
      coderay (>= 1.0.0)
      erubi (>= 1.0.0)
      rack (>= 0.9.0)
    bindex (0.7.0)
    binding_of_caller (0.8.0)
      debug_inspector (>= 0.0.1)
    bootsnap (1.4.4)
      msgpack (~> 1.0)
    builder (3.2.3)
    bullet (6.0.0)
      activesupport (>= 3.0.0)
      uniform_notifier (~> 1.11)
    byebug (11.0.1)
    capybara (3.24.0)
      addressable
      mini_mime (>= 0.1.3)
      nokogiri (~> 1.8)
      rack (>= 1.6.0)
      rack-test (>= 0.6.3)
      regexp_parser (~> 1.5)
      xpath (~> 3.2)
    childprocess (1.0.1)
      rake (< 13.0)
    choice (0.2.0)
    coderay (1.1.2)
    concurrent-ruby (1.1.5)
    connection_pool (2.2.2)
    crass (1.0.4)
    database_cleaner (1.7.0)
    debug_inspector (0.0.3)
    devise (4.6.2)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 4.1.0, < 6.0)
      responders
      warden (~> 1.2.3)
    diff-lcs (1.3)
    docile (1.3.2)
    ebnf (1.1.3)
      rdf (~> 3.0)
      sxp (~> 1.0)
    erubi (1.8.0)
    factory_bot (5.0.2)
      activesupport (>= 4.2.0)
    factory_bot_rails (5.0.2)
      factory_bot (~> 5.0.2)
      railties (>= 4.2.0)
    faker (1.9.3)
      i18n (>= 0.7)
    faraday (0.15.4)
      multipart-post (>= 1.2, < 3)
    ffi (1.11.1)
    friendly_id (5.2.5)
      activerecord (>= 4.0.0)
    fuubar (2.4.0)
      rspec-core (~> 3.0)
      ruby-progressbar (~> 1.4)
    globalid (0.4.2)
      activesupport (>= 4.2.0)
    hamster (3.0.0)
      concurrent-ruby (~> 1.0)
    i18n (1.6.0)
      concurrent-ruby (~> 1.0)
    jaro_winkler (1.5.3)
    jbuilder (2.9.1)
      activesupport (>= 4.2.0)
    jmespath (1.4.0)
    json (2.2.0)
    kaminari (1.1.1)
      activesupport (>= 4.1.0)
      kaminari-actionview (= 1.1.1)
      kaminari-activerecord (= 1.1.1)
      kaminari-core (= 1.1.1)
    kaminari-actionview (1.1.1)
      actionview
      kaminari-core (= 1.1.1)
    kaminari-activerecord (1.1.1)
      activerecord
      kaminari-core (= 1.1.1)
    kaminari-core (1.1.1)
    launchy (2.4.3)
      addressable (~> 2.3)
    link_header (0.0.8)
    listen (3.1.5)
      rb-fsevent (~> 0.9, >= 0.9.4)
      rb-inotify (~> 0.9, >= 0.9.7)
      ruby_dep (~> 1.2)
    loofah (2.2.3)
      crass (~> 1.0.2)
      nokogiri (>= 1.5.9)
    mail (2.7.1)
      mini_mime (>= 0.1.1)
    marcel (0.3.3)
      mimemagic (~> 0.3.2)
    method_source (0.9.2)
    mimemagic (0.3.3)
    mini_magick (4.9.3)
    mini_mime (1.0.1)
    mini_portile2 (2.4.0)
    minitest (5.11.3)
    msgpack (1.3.0)
    multipart-post (2.1.1)
    net-http-persistent (3.0.1)
      connection_pool (~> 2.2)
    nio4r (2.3.1)
    nokogiri (1.10.3)
      mini_portile2 (~> 2.4.0)
    orm_adapter (0.5.0)
    parallel (1.17.0)
    parser (2.6.3.0)
      ast (~> 2.4.0)
    pg (1.1.4)
    pg_search (2.2.0)
      activerecord (>= 4.2)
      activesupport (>= 4.2)
    public_suffix (3.1.0)
    puma (3.12.1)
    pundit (2.0.1)
      activesupport (>= 3.0.0)
    pundit-matchers (1.6.0)
      rspec-rails (>= 3.0.0)
    rack (2.0.7)
    rack-proxy (0.6.5)
      rack
    rack-test (1.1.0)
      rack (>= 1.0, < 3)
    rails (5.2.3)
      actioncable (= 5.2.3)
      actionmailer (= 5.2.3)
      actionpack (= 5.2.3)
      actionview (= 5.2.3)
      activejob (= 5.2.3)
      activemodel (= 5.2.3)
      activerecord (= 5.2.3)
      activestorage (= 5.2.3)
      activesupport (= 5.2.3)
      bundler (>= 1.3.0)
      railties (= 5.2.3)
      sprockets-rails (>= 2.0.0)
    rails-dom-testing (2.0.3)
      activesupport (>= 4.2.0)
      nokogiri (>= 1.6)
    rails-erd (1.6.0)
      activerecord (>= 4.2)
      activesupport (>= 4.2)
      choice (~> 0.2.0)
      ruby-graphviz (~> 1.2)
    rails-html-sanitizer (1.0.4)
      loofah (~> 2.2, >= 2.2.2)
    railties (5.2.3)
      actionpack (= 5.2.3)
      activesupport (= 5.2.3)
      method_source
      rake (>= 0.8.7)
      thor (>= 0.19.0, < 2.0)
    rainbow (3.0.0)
    rake (12.3.2)
    rb-fsevent (0.10.3)
    rb-inotify (0.10.0)
      ffi (~> 1.0)
    rdf (3.0.12)
      hamster (~> 3.0)
      link_header (~> 0.0, >= 0.0.8)
    rdf-aggregate-repo (2.2.1)
      rdf (>= 2.2, < 4.0)
    rdf-xsd (3.0.1)
      rdf (~> 3.0)
    regexp_parser (1.5.1)
    responders (2.4.1)
      actionpack (>= 4.2.0, < 6.0)
      railties (>= 4.2.0, < 6.0)
    rspec-core (3.8.1)
      rspec-support (~> 3.8.0)
    rspec-expectations (3.8.4)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.8.0)
    rspec-mocks (3.8.1)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.8.0)
    rspec-rails (3.8.2)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      railties (>= 3.0)
      rspec-core (~> 3.8.0)
      rspec-expectations (~> 3.8.0)
      rspec-mocks (~> 3.8.0)
      rspec-support (~> 3.8.0)
    rspec-support (3.8.2)
    rubocop (0.71.0)
      jaro_winkler (~> 1.5.1)
      parallel (~> 1.10)
      parser (>= 2.6)
      rainbow (>= 2.2.2, < 4.0)
      ruby-progressbar (~> 1.7)
      unicode-display_width (>= 1.4.0, < 1.7)
    rubocop-performance (1.4.0)
      rubocop (>= 0.71.0)
    rubocop-rails (2.0.1)
      rack (>= 1.1)
      rubocop (>= 0.70.0)
    rubocop-rspec (1.33.0)
      rubocop (>= 0.60.0)
    ruby-graphviz (1.2.4)
    ruby-progressbar (1.10.1)
    ruby_dep (1.5.0)
    rubyzip (1.2.3)
    selenium-webdriver (3.142.3)
      childprocess (>= 0.5, < 2.0)
      rubyzip (~> 1.2, >= 1.2.2)
    sentry-raven (2.9.0)
      faraday (>= 0.7.6, < 1.0)
    shoulda-matchers (4.1.0)
      activesupport (>= 4.2.0)
    simplecov (0.16.1)
      docile (~> 1.1)
      json (>= 1.8, < 3)
      simplecov-html (~> 0.10.0)
    simplecov-html (0.10.2)
    sorbet (0.4.4270)
      sorbet-static (= 0.4.4270)
    sorbet-runtime (0.4.4270)
    sorbet-static (0.4.4270-universal-darwin-14)
    sparql (3.0.2)
      builder (~> 3.2)
      ebnf (~> 1.1)
      rdf (~> 3.0)
      rdf-aggregate-repo (>= 2.2, < 4.0)
      rdf-xsd (~> 3.0)
      sparql-client (~> 3.0)
      sxp (~> 1.0)
    sparql-client (3.0.1)
      net-http-persistent (>= 2.9, < 4)
      rdf (~> 3.0)
    spring (2.1.0)
    spring-watcher-listen (2.0.1)
      listen (>= 2.7, < 4.0)
      spring (>= 1.2, < 3.0)
    sprockets (3.7.2)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.1)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    sxp (1.0.2)
      rdf (~> 3.0)
    thor (0.20.3)
    thread_safe (0.3.6)
    turbolinks (5.2.0)
      turbolinks-source (~> 5.2)
    turbolinks-source (5.2.0)
    tzinfo (1.2.5)
      thread_safe (~> 0.1)
    unicode-display_width (1.6.0)
    uniform_notifier (1.12.1)
    warden (1.2.8)
      rack (>= 2.0.6)
    web-console (3.7.0)
      actionview (>= 5.0)
      activemodel (>= 5.0)
      bindex (>= 0.4.0)
      railties (>= 5.0)
    webdrivers (4.0.1)
      nokogiri (~> 1.6)
      rubyzip (~> 1.0)
      selenium-webdriver (>= 3.0, < 4.0)
    webpacker (4.0.7)
      activesupport (>= 4.2)
      rack-proxy (>= 0.6.1)
      railties (>= 4.2)
    websocket-driver (0.7.1)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.4)
    xpath (3.2.0)
      nokogiri (~> 1.8)

PLATFORMS
  ruby

DEPENDENCIES
  active_storage_validations (~> 0.7)
  addressable (~> 2.6.0)
  aws-sdk-s3 (~> 1.39)
  better_errors (~> 2.5)
  binding_of_caller (~> 0.8)
  bootsnap (>= 1.1.0)
  bullet (~> 6.0)
  byebug
  capybara (>= 2.15)
  database_cleaner (~> 1.7)
  devise (~> 4.6)
  factory_bot_rails (~> 5.0)
  faker (~> 1.9)
  friendly_id (~> 5.2.5)
  fuubar (~> 2.4.0)
  jbuilder (~> 2.9)
  kaminari (~> 1.1)
  launchy (~> 2.4)
  listen (>= 3.0.5, < 3.2)
  mini_magick (~> 4.9)
  pg (>= 0.18, < 2.0)
  pg_search (~> 2.2)
  puma (~> 3.12)
  pundit (~> 2.0)
  pundit-matchers (~> 1.6.0)
  rails (~> 5.2.3)
  rails-erd (~> 1.6)
  rspec-rails (~> 3.8)
  rubocop (~> 0.71)
  rubocop-performance (~> 1.4)
  rubocop-rails (~> 2.0)
  rubocop-rspec (~> 1.33)
  ruby-progressbar (~> 1.10)
  selenium-webdriver (~> 3.142)
  sentry-raven (~> 2.9)
  shoulda-matchers (= 4.1.0)
  simplecov (~> 0.16)
  sorbet (~> 0.4.4270)
  sorbet-runtime
  sparql (~> 3.0.2)
  spring
  spring-watcher-listen (~> 2.0.0)
  turbolinks (~> 5)
  web-console (>= 3.3.0)
  webdrivers (~> 4.0)
  webpacker (~> 4.0.7)

RUBY VERSION
   ruby 2.6.2p47

BUNDLED WITH
   2.0.1
```
</details>

**Comment by vfonic** [comment]
I also came here because I have a `Makefile` in my rails project root after running `srb init`

**Comment by connorshea** [comment]
@vfonic could you share your Gemfile.lock so we can narrow down what’s causing it?

**Comment by vfonic** [comment]
I'll push the whole repo tomorrow (that's in about <10h) and will post here

**Comment by vfonic** [comment]
Hey Jake!

Good to hear it's been fixed. I couldn't find where exactly. So I'm not sure if `Makefile` issue was resolved.

I can repro the `Makefile` being generated on:

```
    sorbet (0.4.4289)
      sorbet-static (= 0.4.4289)
    sorbet-runtime (0.4.4289)
    sorbet-static (0.4.4289-universal-darwin-14)
```

Here are the repro steps:
1. `git clone https://github.com/vfonic/martian-library`
2. `git checkout b64c99d1f16b3c1f4d470d5d5b7ef8d09dca346b` (one commit before last, with no sorbet)
3. 
```sh
echo "
gem 'sorbet', group: :development
gem 'sorbet-runtime'
" >> Gemfile
```
4. `bundle install` (or `bundle update`)
5. `bundle exec srb init`

Expected:

No `Makefile` created.

Actual:

`Makefile` gets created in the project root.

NOTE: I'm using `gem 'rails', '~> 6.0.0.rc1'`. Maybe that could be the reason?

**Comment by connorshea** [comment]
We should be able to narrow down what library is causing the Makefile problem by looking at the Gemfile.lock of my ouija-bot and VideoGameList projects (both exhibit this problem), and comparing it with the Gemfile.lock in yours, and then we can filter it to only the libraries shared by all three.

https://github.com/connorshea/VideoGameList
https://github.com/connorshea/ouija-bot

The exact library versions don’t matter, only the libraries themselves.

**Comment by jaredbeck** [comment]
I spotted this in my `srb init` output:

```
[677/1237] require_relative './node_modules/node-sass/src/libsass/extconf.rb'creating Makefile
```

It might be reasonable for Sorbet to ignore `node_modules`, but that's a can of worms isn't it?

**Comment by vfonic** [comment]
@jaredbeck what do you mean by "that's a can of worms"? It's great you are very eloquent in your English, but please can you explain what you mean, as I don't understand? :)

I think `node_modules` could be ignored for sure. I don't see any reasonable use-case where someone would have ruby source code of their project in that directory.

**Comment by jaredbeck** [comment]
>> It might be reasonable for Sorbet to ignore node_modules, but that's a can of worms isn't it?

> what do you mean by "that's a can of worms"?

Sorry for using an idiom. I meant that the decision would require careful discussion.

**Pull request merged**